src/TLuaInterpreter.cpp

```diff
@@ -12494,7 +12494,14 @@ void TLuaInterpreter::logError( std::string & e, const QString & name, const QSt
         mpHost->mpEditorDialog->mpErrorConsole->printDebug(green, black, s2 );
         mpHost->mpEditorDialog->mpErrorConsole->printDebug(red, black, s3 );
     }
-
+    
+    if( mpHost->mpConsole && mpHost->mEchoLuaErrors )
+    {
+    	  blue = QColor(80,80,255);
+        mpHost->mpConsole->printDebug(blue, black, QString("[ ERROR ] - ") );
```

Comment:
It _might_ be worth considering a new _type_ for these - possibly something like
    "[  LUA  ]  - \<error message from lua sub-system\>"
and we could make it blue if you really want it to be!

OTOH I can't be certain that all the possible sources of messages from the Lua sub-system will fit into this style AND I'm not sure if ALL the messages are error ones, it might be worth experimenting to see if various "OK" messages come from there as well...